include/tvm/arith/analyzer.h

```diff
@@ -411,8 +412,9 @@ class TVM_DLL Analyzer {
    *
    * \param var The variable.
    * \param expr The expression we bind to.
+   * \param override Whether do we allow override of existing information.
```

Comment:
```suggestion
   * \param override Whether do we allow override of existing binding variables.
```
suggest to be more specific.

Reply:
Rephrasing the comments.  Thanks